Pipfile

```diff
@@ -102,7 +102,7 @@ marshmallow = ">=2.15.1,<3.0.0"
 pydocstyle = ">=3.0.0"
 pytest = ">=4.6.4"
 pytest-cov = ">=2.7.1"
-pytest-invenio = ">=1.2.1,<1.3.0"
+pytest-invenio = ">=1.3.0"
```

Comment:
This can probably be removed.

Reply:
Do you talk about the entire line or simply remove the constraint on the version?

In the first case I don't understand how we can use pytest-invenio if no other package depends on it.
In the second case, I'm not sure that it will be safe to let pipenv choose the version as the 1.2.2 is problematic for us.

I think that's the way I do is the best case in term of dependancies.

For your information, that was my ideas before this PR: 

```
pytest-invenio = "==1.2.1"
pytest-invenio = ">=1.3.0"
pytest-invenio = ">=1.2.1,!=1.2.2"
```